tests/ha/single_pod_restart/test_data_server_pod_restart.py

```diff
@@ -518,3 +534,192 @@ def test_io_data_server_pod_restart(self):
+        wr_resp = tuple()
+        while len(wr_resp) != 3:
+            wr_resp = wr_output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        buckets = s3_test_obj.bucket_list()[1]
+        assert_utils.assert_equal(len(buckets), wr_bucket, f"Failed to create {wr_bucket} number "
+                                                           f"of buckets. Created {len(buckets)} "
+                                                           "number of buckets")
+        s3_data = wr_resp[0]
+        LOGGER.info("Step 3.1: Successfully performed WRITEs with variable object sizes on %s "
+                    "buckets for parallel DELETEs.", wr_bucket)
+        LOGGER.info("Step 3.2: Perform WRITEs with variable object sizes for parallel READs")
+        test_prefix_read = f'test-45517-read-{int(perf_counter_ns())}'
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users_org.values())[0],
+                                                    log_prefix=test_prefix_read, skipread=True,
+                                                    skipcleanup=True, nclients=5, nsamples=5,
+                                                    setup_s3bench=False)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 3.2: Performed WRITEs with variable sizes objects for parallel READs.")
+        LOGGER.info("Step 4: Starting three independent background threads for READs, WRITEs & "
+                    "DELETEs.")
+        LOGGER.info("Step 4.1: Start continuous DELETEs in background on random %s buckets",
+                    del_bucket)
+        bucket_list = s3_data.keys()
+        get_random_buck = self.system_random.sample(bucket_list, del_bucket)
+        args = {'test_prefix': test_prefix_del, 'test_dir_path': self.test_dir_path,
+                'skipput': True, 'skipget': True, 'bkt_list': get_random_buck, 'output': del_output}
+        thread_del = threading.Thread(target=self.ha_obj.put_get_delete,
+                                      args=(event, s3_test_obj,), kwargs=args)
+        thread_del.daemon = True  # Daemonize thread
+        LOGGER.info("Step 4.1: Successfully started DELETEs in background for %s buckets",
+                    del_bucket)
+        LOGGER.info("Step 4.2: Perform WRITEs with variable object sizes in background")
+        test_prefix_write = f'test-45517-write-{int(perf_counter_ns())}'
+        output_wr = Queue()
+        args = {'s3userinfo': list(users_org.values())[0], 'log_prefix': test_prefix_write,
+                'nclients': 1, 'nsamples': 5, 'skipread': True, 'skipcleanup': True,
+                'output': output_wr, 'setup_s3bench': False}
+        thread_wri = threading.Thread(target=self.ha_obj.event_s3_operation, args=(event,),
+                                      kwargs=args)
+        thread_wri.daemon = True  # Daemonize thread
+        LOGGER.info("Step 4.2: Successfully started WRITEs with variable sizes objects in "
+                    "background")
```

Comment:
We need to add DTM flag for both WRITE and DELETE threads.
If DTM is disabled then only take different prefix otherwise you ll have to take same prefix as healthy mode.
And for DELETE, if DTM is disabled then only start delete thread.